### PR TITLE
GP2-950-scripts-ep-json-fields

### PR DESCRIPTION
**Pull request by depsiatwal**
- [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added.

### CODE REVIEW

**Comment by depsiatwal** [comment]
Not the most exhaustive tests just wanted to test one of the offending fields but will make more comprehensive later. Will also add validator to the model to prevent scripts entering the DB.

File: exportplan/serializers.py [hunk 1]
```diff
@@ -79,9 +80,9 @@ def validate_target_markets(self, values):
...
 
 class CompanyObjectiveSerializer(serializers.Serializer):
-    description = serializers.CharField(required=False, allow_blank=True)
-    planned_reviews = serializers.CharField(required=False, allow_blank=True)
-    owner = serializers.CharField(required=False, allow_blank=True)
+    description = serializers.CharField(required=False, allow_blank=True, validators=[no_html])
+    planned_reviews = serializers.CharField(required=False, allow_blank=True, validators=[no_html])
+    owner = serializers.CharField(required=False, allow_blank=True, validators=[no_html])
     start_date = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     end_date = serializers.CharField(required=False, allow_blank=True, allow_null=True)
```
Comment (stevejalim) [inline exportplan/serializers.py:87]: Should `start_date` and `end_date` also have the `no_html` validator?

**Review by stevejalim — COMMENTED** [review]
Looks good - one question

File: exportplan/serializers.py (hunk 1 above)
Comment (depsiatwal) [reply exportplan/serializers.py]: good point won't do any harm

**Review by stevejalim — APPROVED** [review]
LGTM ❤️

**Pull request merged**